commands/system/update.sh

```diff
@@ -10,6 +10,14 @@ function system:update() {
 
     _logYellow "Updating dde repository"
     cd ${ROOT_DIR}
+    branch=$(git rev-parse --abbrev-ref HEAD)
+    if [ $branch != 'master' ] && [ $branch != 'main' ]
+    then
+        _logYellow "#"
+        _logYellow "# Be careful!"
+        _logYellow "# You have not checked out master. You're currently on: $branch"
```

Comment:
@Ruesa18 1 log should be sufficient (like all other places we warn with _logYellow). If we need additional formatting for warnings, we should add it to the _logYellow function